classes/fields/number.php

```diff
@@ -137,68 +137,76 @@ public function options () {
         return $options;
     }
 
-    /**
-     * Define the current field's schema for DB table storage
-     *
-     * @param array $options
-     *
-     * @return array
-     * @since 2.0
-     */
-    public function schema ( $options = null ) {
-        $length = (int) pods_var( self::$type . '_max_length', $options, 12, null, true );
+	/**
+	 * Define the current field's schema for DB table storage
+	 *
+	 * @param array $options
+	 *
+	 * @return string
+	 * @since 2.0
+	 */
+	public function schema( $options = null ) {
 
-        if ( $length < 1 || 64 < $length )
-            $length = 64;
+		$length = (int) pods_v( static::$type . '_max_length', $options, 12, true );
```

Comment:
Thank you for the pods_v stuff here, it helps.

We call pods_var in so many places, it has an extra sanitization call that's not needed in many cases. So converting to pods_v will help us long term too.